interface/CH.json

```diff
@@ -2248,20 +2248,20 @@
         }
       }
     },
-    "/IdentityProviders/{id1}/Anvenderkontekster/{id2}": {
+    "/IdentityProviders/{id}/Anvenderkontekster/{cvr}": {
```

Comment:
Please avoid unnecessarily changes of the interface. We will need approval from our customers for all changes that we do.